schema.sql

```diff
@@ -49,6 +49,13 @@ CREATE TABLE specialization (
 CREATE TABLE visits (
     id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
     animals_id INT REFERENCES animals(id),
-    vet_id INT REFERENCES vets(id) 
+    vet_id INT REFERENCES vets(id)
     date_of_visit DATE NOT NULL
 );
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX animal_index ON visits(animal_id);
+CREATE INDEX vet_index ON visits(vet_id);
+CREATE INDEX email_index ON owners(email);
```

Comment:
- [ ] @techwolf404 and @29td great work on your queries but you need to include before and after screenshots of every query so as to showcase the improvement in execution time.
- Kindly strive to achieve less than 1 sec execution time in your after query (nb.  its not a must to achieve it but its highly recommended)